client/src/components/AddUserForm.tsx

```diff
@@ -1,25 +1,43 @@
 import React, { useState, useEffect } from 'react';
 import Button from 'plaid-threads/Button';
 import TextInput from 'plaid-threads/TextInput';
+import Checkbox from 'plaid-threads/Checkbox';
 
 import { useUsers, useCurrentUser } from '../services';
 
+const PLAID_ENV = process.env.REACT_APP_PLAID_ENV;
+
 interface Props {
   hideForm: () => void;
 }
 const AddUserForm = (props: Props) => {
   const [username, setUsername] = useState('');
   const [fullname, setFullname] = useState('');
   const [email, setEmail] = useState('');
+  const [verifyIdentity, setVerifyIdentity] = useState(false);
```

Comment:
I think this could be named a bit more intuitively. Is this representing if we should verify the identity? Or does this represent whether the identity is verified?

I.e. we could rename it to either `isIdentityVerified` or `shouldVerifyIdentity`

Reply:
good call. thanks